### ISSUE TITLE
[1.12] Places liquid from a clay bucket makes the bucket become an iron bucket?

### ISSUE THREAD

**Issue by NoName1245** [KnightMiner/Ceramics#14]
I was doing my everyday messing-around-with-mods-in-creative routine, and when I when I was looking at Ceramics features, pouring out liquid made the bucket become iron (only works in Survival, had discovered this because I used /gamemode 0 to test a few things with damage from another mod) and thus Creative will not have this issue). I have tested this with only Ceramics and Forge in a new instance, so it not caused by another mod. 

TLDR: Clay buckets with liquid become iron when emptied.

**Comment by KnightMiner** [comment]
Seems Forge changed something in the way its universal bucket works. I fixed this for the next build
